operator/cmd/syncclusterconfig/sync.go

```diff
@@ -276,6 +281,7 @@ type Syncer struct {
 type ClusterConfigStatus struct {
 	Version      int64
 	NeedsRestart bool
+	Hash         string
```

Comment:
Can we give this a more descriptive name and/or a comment? `Hash` is quite misleading.